resources/compass/charts/director/templates/deployment.yaml

```diff
@@ -61,7 +61,7 @@ spec:
               valueFrom:
                 secretKeyRef:
                   name: compass-postgresql
-                  key: postgresql-directorDatabaseName
+                  key: postgresql-director-name
```

Comment:
maybe better will be: `postgresql-director-db-name`? The same in other places when we define db name